### PR TITLE
User Email Check on Signup

### LINKED ISSUE #1992: Checking If Email exists in realtime during signup
**I'm submitting a ...**
- [x] feature request

**Current behaviour:**

Currently , user need to enter email and it then send to backend for checking if user exists .
**Expected behaviour:**- [ ] bug report

Checking User in realtime.

**Steps to reproduce:**

**Screenshots**

**Would you like to work on the issue?**

Yes.

### PR DESCRIPTION
**Pull request by sk9331657**
Fixes #1992 

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] I have added necessary documentation (if appropriate)

#### Changes proposed in this pull request:

- Existing User Email Check on Signup

### Screenshots for the change
ezgif com-video-to-gif 10

### CODE REVIEW

**Comment by sk9331657** [comment]
@fossasia/badgeyay-admins  Please review.

File: frontend/app/templates/components/forms/signup-form.hbs [hunk 1]
```diff
@@ -10,9 +10,11 @@
                 <div class="field required">
                     <div class="ui left icon input">
                         <i class="mail icon"></i>
-                        {{input type="text" value=email name="email" placeholder="E-mail address"}}
+                        {{input type="text" value=email name="email" id="email" keyUp=(action 'emailcheck') placeholder="E-mail address"}}
```
Comment (abishekvashok) [inline frontend/app/templates/components/forms/signup-form.hbs]: Prefer keyPress than keyUp which doesn't trigger when backspace is pressed.

File: frontend/app/templates/components/forms/signup-form.hbs [hunk 2]
```diff
@@ -10,9 +10,11 @@
...
                     </div>
+                    <div id="emailstatus" style="text-align:center"></div>
```
Comment (abishekvashok) [inline frontend/app/templates/components/forms/signup-form.hbs]: Please don't use inline css

**Comment by mariobehling** [comment]
Closing this due to inactivity. Travis not working.

**Pull request closed without merging**